buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt

```diff
@@ -132,27 +134,25 @@ object Dependencies {
     const val kotlinGradlePlugin = "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
     const val kotlinReflect = "org.jetbrains.kotlin:kotlin-reflect:$KOTLIN_VERSION"
     const val kotlinStdLib = "org.jetbrains.kotlin:kotlin-stdlib:$KOTLIN_VERSION"
-    const val ktlintPlugin = "org.jlleitschuh.gradle:ktlint-gradle:$KTLINT_VERSION"
+    const val ktlintPlugin = "org.jlleitschuh.gradle:ktlint-gradle:$KTLINT_PLUGIN_VERSION"
     const val leakCanary = "com.squareup.leakcanary:leakcanary-android:$LEAK_CANARY_VERSION"
-    const val loggingInterceptor = "com.squareup.okhttp3:logging-interceptor:$OKHTTP_VERSION"
     const val materialComponents = "com.google.android.material:material:$MATERIAL_COMPONENTS_VERSION"
-    const val marwonCore = "io.noties.markwon:core:$MARWON_VERSION"
-    const val marwonLinkify = "io.noties.markwon:linkify:$MARWON_VERSION"
-    const val marwonextStrikethrough = "io.noties.markwon:ext-strikethrough:$MARWON_VERSION"
-    const val marwonImage = "io.noties.markwon:image:$MARWON_VERSION"
-    const val mockito = "org.mockito:mockito-core:$MOKITO_VERSION"
-    const val mockitoKotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:$MOKITO_KOTLIN_VERSION"
+    const val markwonCore = "io.noties.markwon:core:$MARKWON_VERSION"
+    const val markwonLinkify = "io.noties.markwon:linkify:$MARKWON_VERSION"
+    const val markwonextStrikethrough = "io.noties.markwon:ext-strikethrough:$MARKWON_VERSION"
+    const val markwonImage = "io.noties.markwon:image:$MARKWON_VERSION"
+    const val mockito = "org.mockito:mockito-core:$MOCKITO_VERSION"
+    const val mockitoKotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:$MOCKITO_KOTLIN_VERSION"
     const val navigationFragmentKTX = "androidx.navigation:navigation-fragment-ktx:$NAVIGATION_VERSION"
     const val navigationSafeArgsGradlePlugin = "androidx.navigation:navigation-safe-args-gradle-plugin:$NAVIGATION_VERSION"
     const val navigationRuntimeKTX = "androidx.navigation:navigation-runtime-ktx:$NAVIGATION_VERSION"
     const val navigationUIKTX = "androidx.navigation:navigation-ui-ktx:$NAVIGATION_VERSION"
     const val ok2curl = "com.github.mrmike:ok2curl:$OK2CURL_VERSION"
-    const val okhttp = "com.squareup.okhttp3:okhttp:$OKHTTP_VERSION"
-    const val okhttpLoggingInterceptor = "com.squareup.okhttp3:logging-interceptor:$OKHTTP_VERSION"
-    const val okhttpMockWebserver = "com.squareup.okhttp3:mockwebserver:$OKHTTP_VERSION"
+    const val okhttp = "com.squareup.okhttp3:okhttp"
+    const val okhttpBom = "com.squareup.okhttp3:okhttp-bom:$OKHTTP_BOM_VERSION"
```

Comment:
Same as with Firebase, started using the BoM for OkHttp too, as they do publish one